tumblr-importer.php

```diff
@@ -819,6 +819,10 @@ function fetch_posts($url, $start=0, $count = 50, $email = null, $password = nul
 		switch ( $response->meta->status ) {
 			case 200: // OK
 				break;
+			case 420: // rate limited
+			case 429: // rate limited
+				// make this chill for a minute
+				return false;
```

Comment:
looking at the calling side(s), returning `false` seems to make it think it just needs to wait for later? it's hard for me to understand how this is meant to work, and it's definitely not resilient for this kind of event. looks like it'll keep `while` looping and getting a `429 Limit Exceeded`, but maybe at least now it won't fail outright and stop trying.